packages/editor-ui/src/components/ActivationModal.vue

```diff
@@ -88,7 +89,7 @@ export default defineComponent({
 		},
 		handleCheckboxChange(checkboxValue: boolean) {
 			this.checked = checkboxValue;
-			window.localStorage.setItem(LOCAL_STORAGE_ACTIVATION_FLAG, checkboxValue.toString());
+			useStorage(LOCAL_STORAGE_ACTIVATION_FLAG).value = checkboxValue.toString();
```

Comment:
same here. we could add the ref in `setup`
```typescript
	setup() {
		return {
			activationFlag: useStorage(LOCAL_STORAGE_ACTIVATION_FLAG),
		};
	},
```

and then update this to 
```suggestion
			this.activationFlag.value = checkboxValue.toString();
```

Reply:
since it's not reused, will leave as is for now.